### PR TITLE
adding ssh secret template

### PR DESCRIPTION
**Pull request by mchengat-zz**
## Context

 Add sshca private to kubernetes secret 

## Objective

Add sshca private to kubernetes secret 

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tk3fftk** [comment]
I'm not sure the purpose. but that file looks misunderstandable. 
And If it's optional resource, it should be opt-in or opt-out in chart practice.

**Comment by mchengat-zz** [comment]
> opt-in or opt-out

Could you please give some clarification about what you mean by opt-in or opt-out

**Comment by tk3fftk** [comment]
It's kind of feature flag like notification setting in screwdriver chart. 
https://github.com/screwdriver-cd/screwdriver-chart/blob/84c798a4959d5284117d3adb462efdd8b97b952d/templates/api.yaml#L168-L174

Users have to prepare some files for the secret even if user don't need the secret if the secret creation is not opt-out.

File: templates/secret.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sd-secret
+  namespace: {{ .Values.namespace }}
+  annotations:
+    kubernetes.io/service-account.name: sdsecret
+type: Opaque
+data:
+{{ (.Files.Glob "key/*").AsSecrets | indent 2 }}
```
Comment (jithine) [inline templates/secret.yaml]: is key the folder the secrets reside ? maybe rename it as `secrets`
Is this supposed to hold one secrte per file ? How is the secret name handled ?

**Pull request closed without merging**